bin/setup.sh

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env bash
 
-# Check for Carthage
-printf "Checking for Carthage: "
-carthage version >/dev/null 2>&1 || { printf >&2 "nope!\nCarthage must be installed. Try running:\n\tbrew install carthage\n"; exit 1; }
-printf "found!\n"
+# Check Xcode version (TODO: semver)
+printf "Checking Xcode version: "
+[ `xcodebuild -version | grep Xcode | awk '{print $2}'` = "9.0" ] || { printf >&2 "nope!\nXcode 9.0 must be installed\n"; exit 1; }
```

Comment:
This, unfortunately, doesn't work for two reasons:
 - It only reports the command-line tools version for the selected instance, of which there are many possibilities, so this could fail for someone who has Xcode 8.3 and Xcode 9 installed simultaneously, and
 - it doesn't take into account future versions of Xcode.

As for the first problem, this could probably be downgraded to an informative warning rather than an error.

I'm less sure how to fix the second problem. String ordering would work for versions until we hit Xcode 10. My bash-fu is not strong in this area. Any ideas?